frontends/ol-search-ui/src/components/ShareTooltip.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useRef } from "react"
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from "react-share"
+import { FacebookIcon, TwitterIcon, LinkedinIcon } from "react-share"
+import Tooltip from "rc-tooltip"
+
+type HelperProps = {
+  url: string
+  hideSocialButtons?: boolean | null | undefined
+  objectType?: string | null | undefined
+}
+
+export const ShareTooltipHelper: React.FC<HelperProps> = ({
+  url,
+  hideSocialButtons,
+  objectType
+}) => {
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const selectAndCopyLinkText = (e: React.MouseEvent<HTMLElement>) => {
+    e.preventDefault()
+    if (inputRef.current) {
+      navigator.clipboard.writeText(inputRef.current.value)
+    }
+  }
+
+  return (
+    <div className="tooltip">
+      <div className="tooltip-text">
+        {`Share a link to this ${objectType}:`}
+      </div>
```

Comment:
This should probably be an actual button, although I'm not sure the tooltip is keyboard focusable at all right now and that would probably take a little more effort. (Possibly best done by finding a different library that handles it for us.)

Reply:
Good catch that the tooltip isn't tab operable. I'm going to fix this in a followup pr